README.md

```diff
@@ -3,10 +3,78 @@
 ![build](https://github.com/endjin/dependabot-pr-parser/workflows/build/badge.svg)
 [![GitHub license](https://img.shields.io/badge/License-Apache%202-blue.svg)](https://raw.githubusercontent.com/corvus-dotnet/Corvus.Deployment/master/LICENSE)
 
+This repo contains 3 docker-based GitHub actions that can be used to enable continuous integration for a configurable subset of dependencies being managed by GitHub Dependabot.
+
+## Overview
+
+Whilst tools like [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) simplify the process of testing updates to a repository's package dependencies, there can still be signficant coordination work to apply such changes and have the resulting updates cascade through the rest of your software estate.
+
+The GitHub actions detailed below provide the plumbing to enable the following features:
+1. The ability to identify which dependencies are being updated and the 'scale' of the update, in terms of the [semantic version](https://semver.org) increment
+1. A policy-based approach for managing which dependencies are approved for being automatically merged
+1. The ability to batch multiple, auto-mergeable dependency updates into a single release 'event'
+
```

Comment:
Could you include the workflow diagram in the repo, so it can be displayed in the readme?